packages/capabilities/src/blob.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Blob Capabilities.
+ *
+ * Blob is a fixed size byte array addressed by the multihash.
+ * Usually blobs are used to represent set of IPLD blocks at different byte ranges.
+ *
+ * These can be imported directly with:
+ * ```js
+ * import * as Blob from '@web3-storage/capabilities/blob'
+ * ```
+ *
+ * @module
+ */
+import { capability, Schema } from '@ucanto/validator'
+import { equalBlob, equalWith, SpaceDID } from './utils.js'
+
+/**
+ * Agent capabilities for Blob protocol
+ */
+
+/**
+ * Capability can only be delegated (but not invoked) allowing audience to
+ * derived any `blob/` prefixed capability for the (memory) space identified
+ * by DID in the `with` field.
+ */
+export const blob = capability({
+  can: 'blob/*',
+  /**
+   * DID of the (memory) space where Blob is intended to
+   * be stored.
+   */
+  with: SpaceDID,
+  derives: equalWith,
+})
+
+/**
+ * Blob description for being ingested by the service.
+ */
+export const blobStruct = Schema.struct({
+  /**
+   * A multihash digest of the blob payload bytes, uniquely identifying blob.
+   */
+  content: Schema.bytes(),
```

Comment:
```suggestion
  digest: Schema.bytes(),
```

Reply:
Per the spec this is  `content` https://github.com/web3-storage/specs/blob/main/w3-blob.md#blob-content. Would you prefer to change it?

Reply:
Yeah I was thinking of changing it to align with [`Content-Digest` header](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-digest-headers)

Reply:
I was working on this https://github.com/web3-storage/specs/pull/118/files and forgot to mention